v3/CHANGELOG.md

```diff
@@ -1,4 +1,31 @@
 # Changelog
+## Version 3.0.0-pre.1089 - Jan 17, 2023
+
+Version 3.0.0-pre.1089 has some bug fixes with accessibility that were found by Allyant and VoiceOver and features related to enabling free component layouts, slider animation, graph brush palette, etc.
+
+### Features/Improvements
+- #26: This table of editable cells lacks instructions for screen reader users [#183939669](https://www.pivotaltracker.com/story/show/183939669)
+- A numeric attribute used as a legend produces a choropleth legend [#184029172](https://www.pivotaltracker.com/story/show/184029172)
+- Add functionality to change points properties and graph background properties [#183929138](https://www.pivotaltracker.com/story/show/183929138)
+- Slider animation [#184098342](https://www.pivotaltracker.com/story/show/184098342)
+- Graph brush palette allows categories to be assigned colors [#183987088](https://www.pivotaltracker.com/story/show/183987088)
+- A generic Component exists that serves as a container for any type of component [#183037541](https://www.pivotaltracker.com/story/show/183037541)
+- Enable free layout of components (rather than mosaic/dashboard) [#184210630](https://www.pivotaltracker.com/story/show/184210630)
+- Slider should use SliderModel prop rather than default SliderModel [#184210554](https://www.pivotaltracker.com/story/show/184210554)
+- Graph should use GraphModel prop rather than default GraphModel [#184210550](https://www.pivotaltracker.com/story/show/184210550)
+
+### Bug Fixes
+- Incrementing build numbers fails [#184154683](https://www.pivotaltracker.com/story/show/184154683)
+- #30: Additional content is only revealed on hover, which is inaccessible to screen reader and keyboard users [#183939655](https://www.pivotaltracker.com/story/show/183939655)
+- #25: Keyboard users cannot trigger and navigate these menus [#183939697](https://www.pivotaltracker.com/story/show/183939697)
```

Comment:
```suggestion
- `#30`: Additional content is only revealed on hover, which is inaccessible to screen reader and keyboard users [#183939655](https://www.pivotaltracker.com/story/show/183939655)
- `#25`: Keyboard users cannot trigger and navigate these menus [#183939697](https://www.pivotaltracker.com/story/show/183939697)
```
Otherwise, GitHub is treating these like PR references and the GitHub UI is presenting them as though they're related to those otherwise unrelated PRs (undo/redo in this case).